wgengine/monitor/monitor_windows.go

```diff
@@ -198,8 +205,14 @@ func notifyRouteChange(h *windows.Handle, o *windows.Overlapped) error {
 	return callNotifyProc(notifyRouteChangeProc, h, o)
 }
 
+// forceOverlapEscape exists purely so we can assign to it
+// and make sure that callNotifyProc's 'o' argument does not
+// stay stack allocated.
+var forceOverlapEscape atomic.Value // of *windows.Overlapped
```

Comment:
Using an atomic.Value for this is nice. I've not seen that before.

Why do this for `o` and not for `h` (`*windows.Handle`)?